addons/mail/static/src/models/attachment_list/attachment_list.js

```diff
@@ -64,6 +64,13 @@ registerModel({
             compute: '_computeAttachments',
             inverse: 'attachmentLists',
         }),
+        /**
+         * Link with a AttachmentBoxView to handle attachments.
+         */
+        attachmentBoxView: one('AttachmentBoxView', {
```

Comment:
rename to `attachmentBoxViewOwner` to follow a newish "guideline" we have for parent fields in view models

Reply:
Done !